cgi/generate_sample_import_file.pl

```diff
@@ -59,13 +59,35 @@
 	mandatory => $workbook->add_format(border => 1, bold => 1, bg_color => '#aaffcc'),
 	recommended => $workbook->add_format(border => 1, bold => 1, bg_color => '#ccffdd'),
 	optional => $workbook->add_format(border => 1, bold => 1, bg_color => '#eeffee'),
+	example => $workbook->add_format(italic => 1, valign => 'vcenter', text_wrap => 1,),
 );
 
 # Re-use the structure used to output select2 options in import_file_select_format.pl
 my $select2_options_ref = generate_import_export_columns_groups_for_select2([$lc]);
 
 my $headers_row = 0;
-my $col = 0;
+my $col = 1;
+my $example_row = 2;
+
+my $example = lang("example");
+
+my $example_tsv_file = 'conf/pro-platform/Import template - Example translations - Import sheet.tsv';
+my %example_values_by_header;
+
+open(my $example_fh, '<', $example_tsv_file) or die "Cannot open $example_tsv_file: $!";
+my $header_line = <$example_fh>;
+chomp($header_line);
+my @headers = split("\t", $header_line);
+while (my $line = <$example_fh>) {
+    chomp($line);
+    my @values = split("\t", $line);
+    for my $i (0 .. $#headers) {
+        next if $headers[$i] eq 'lc';
+        $example_values_by_header{$headers[$i]} = $values[$i] if defined $values[$i];
+    }
+    last;
```

Comment:
Since you have "last;", you are only reading one line (the first one: the one for English)

Reply:
You should read all the lines, and populate a structure that contains example values for different languages. First you read the "lc" into $line_lc, and then you can create a structure $example_values_by_language_and_header{$line_lc} = {}, and you populate the headers in it.